test/pricing.test.js

```diff
@@ -105,7 +105,7 @@ describe('Recurly.Pricing', function () {
         assert.equal(price.taxes[0].type, 'vat');
         assert.equal(price.taxes[0].region, 'GB');
         assert.equal(price.taxes[0].rate, '0.2');
-        assert.equal(price.now.tax, '0.40');
+        assert.equal(price.now.tax, '4.40');
```

Comment:
These values had to change to respect the more basic behaviors of the `basic` plan (no trial period).